src/components/Blocking/BlockProfile.js

```diff
@@ -78,7 +79,7 @@ const styles = StyleSheet.create({
   title: {
     fontFamily: fonts.inter[600],
     fontSize: 18,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value has been replaced with a reference to the `COLORS` object. This is a good practice as it improves consistency and maintainability of the code by centralizing the color definitions.
```diff
-    color: '#000',
+    color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->